InnerEye/ML/Histopathology/utils/heatmap_utils.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import numpy as np
+
+from matplotlib.axes import Axes
+from matplotlib.image import AxesImage
+import matplotlib.patches as patches 
+import matplotlib.collections as collection
+
+
+def plot_heatmap_selected_tiles(tile_coords: np.array,
```

Comment:
afaik, the correct type annotation is np.ndarray, not np.array

Reply:
(here and below)

Reply:
Changed this, thanks